### PR TITLE
Return headers from get_url

### PR DESCRIPTION
**Pull request by jeanconn**
It is handy to be able to track content modification times via
the last-modified header.

### CODE REVIEW

**Comment by taldcroft** [comment]
Are there external dependencies on `get_url` that will get broken?  Or does perl silently drop off the extra return arg for old code?

**Comment by jeanconn** [comment]
Per your checking (and my separate checking) Perl silently drops the extra return args, so this is back compatible.  I've explicitly run current Replan Central against this in my dev ska and not seen any errors.

File: lib/Ska/Web.pm [hunk 1]
```diff
@@ -121,10 +121,11 @@ sub get_html_content {
 	my $uri = URI->new($src);
 	my $img_uri = $uri->abs($base_uri);
 	next if $img_uri =~ /$dummy_url/; # No base URL supplied and src is relative
-	my ($image_data, $error) = get_url($img_uri);
+	my ($image_data, $error, $header) = get_url($img_uri);
 	push @images, {data => $image_data,
 		       name => $img_uri->rel($img_uri)->as_string,
 		       url  => $img_uri->as_string,
+                       header => $header,
 		      } unless defined $error;
```
Comment (taldcroft) [inline lib/Ska/Web.pm:129]: Unrelated, but having been away from Perl for a while it seems like `unless` is a really bad language construct, esp. in the context of a long-ish code statement like this.  I was looking at this for a bit wondering why `$error` was being ignored.

Comment (jeanconn) [reply lib/Ska/Web.pm:129]: Yeah, I think Conway votes against `unless` and against multi line postfix .   But I'm not sure if this warrants a low-priority issue.

Comment (taldcroft) [reply lib/Ska/Web.pm:129]: No this does not need fixing.

**Pull request merged**